test/e2e/autoscaling/cluster_autoscaler_scalability.go

```diff
@@ -349,7 +349,7 @@ var _ = framework.KubeDescribe("Cluster size autoscaler scalability [Slow]", fun
 		defer framework.DeleteRCAndWaitForGC(f.ClientSet, f.Namespace.Name, podsConfig.Name)
 
 		// Ensure that no new nodes have been added so far.
-		gomega.Expect(e2enode.TotalReady(f.ClientSet)).To(gomega.Equal(nodeCount))
+		framework.ExpectEqual(e2enode.TotalReady(f.ClientSet), nodeCount)
```

Comment:
The test jobs are failed due to
```
ERROR(all) test/e2e/autoscaling/cluster_autoscaler_scalability.go:352:25: 2-valued e2enode.TotalReady(f.ClientSet) (value of type (int, error)) where single value is expected
```
So the TotalReady() returns (int, error) as
```
func TotalReady(c clientset.Interface) (int, error) {
```
I am not sure how current code can pass the existing test jobs.
Anyways these jobs can be successful if we change here like:
```
count, _ := e2enode.TotalReady(f.ClientSet)
framework.ExpectEqual(count, nodeCount)
```

Reply:
Already force pushed. The current function `gomega.Expect` accept multi-parameter. 
Sorry  for my carelessness :)